apm-sniffer/apm-sdk-plugin/mongodb-3.x-plugin/src/main/java/org/skywalking/apm/plugin/mongodb/v3/MongoDBMethodInterceptor.java

```diff
@@ -200,11 +200,18 @@ private String limitFilter(String filter) {
     public void onConstruct(EnhancedInstance objInst, Object[] allArguments) {
         Cluster cluster = (Cluster)allArguments[0];
         StringBuilder peers = new StringBuilder();
-        for (ServerDescription description : cluster.getDescription().getServerDescriptions()) {
+        List<ServerDescription> servers;
+        try {
+            cluster.getDescription().getClass().getMethod("getServerDescriptions", null);
```

Comment:
Is that code need call?

Reply:
The method "getServerDescriptions“ is  V3.3.0 new method, the previous version of the call will be reported null pointer, in order to support the previous version so need to be judged. a comment